manual-tests/automated-test.py

```diff
@@ -0,0 +1,342 @@
+            print("[Info] test runtime: SUCCESS.")
+            print(f"[Info] UUID runtime: {runtimes_string[i]}")
+            zf_conf.uuid_runtimes = runtimes_string[i]
+    if zf_conf.uuid_runtimes is None:
+        print("[Info] test runtime: FAILED.")
+
+    print(f'[Info] Launching Zenoh-Flow example: {zf_command["dataflow_yaml_cmd"]}')
+
+    launch_zf_yaml = os.popen(zf_command["dataflow_yaml_cmd"])
+
+    zf_example_string = launch_zf_yaml.read()
+    print(f"[Info] zf_example_string: {zf_example_string}")
+    if zf_example_string.__len__() == 0:
+        print("[Error] Zenoh-Flow example: FAILED.")
+    else:
+        zf_conf.uuid_runtimes = zf_example_string
+        print(f"[Info] Zenoh-Flow example UUID: {zf_conf.uuid_runtimes}")
+
+    print(f'[Info] Launching list instances: {zf_command["instances_cmd"]}')
+    launch_instances = os.popen(zf_command["instances_cmd"])
+    count = 0
+    instances_string = launch_instances.read().split()
+    for i in range(instances_string.__len__()):
+        if instances_string[i][0] == "+":
+            count = count + 1
+    if count <= 2:
+        print("[Error] test data flow: FAILED.")
+        exit()
+    else:
+        print("[Info] test data flow instance: SUCCESS.")
+        instance["uuid_instances"] = instances_string[23]
+        instance["flow"] = instances_string[25]
+        instance["operators"] = instances_string[27]
+        instance["sinks"] = instances_string[29]
+        instance["sources"] = instances_string[31]
+        instance["connectors"] = instances_string[33]
+        instance["links"] = instances_string[35]
+else:
+    print("[Error] test zenohd process: FAILED.")
+
+
+with open(data_flow_path, "r") as f:
+    try:
+        data = yaml.load(f, Loader=SafeLoader)
+    except yaml.YAMLError as e:
+        print(f"[Error] {e}")
+
+    if str(tuple(data["sources"]).__len__()) == instance["sources"]:
+        print(
+            f'[Info] test active Source nodes: SUCCESS, the active Source nodes [{str(tuple(data["sources"]).__len__())}] are equal to declared Source nodes [{instance["sources"]}]'
+        )
+    if str(tuple(data["sinks"]).__len__()) == instance["sinks"]:
+        print(
+            f'[Info] test active Sink nodes: SUCCESS, the active Sink nodes [{str(tuple(data["sinks"]).__len__())}] are equal to those declared Sinks nodes [{instance["sinks"]}]'
+        )
+    if str(tuple(data["operators"]).__len__()) == instance["operators"]:
+        print(
+            f'[Info] test active Operator nodes: SUCCESS, the active Operator nodes [{str(tuple(data["operators"]).__len__())}] are equal declared Operator nodes [{instance["operators"]}]'
+        )
+    if (
+        str(tuple(data["sources"]).__len__()) != instance["sources"]
+        or str(tuple(data["sinks"]).__len__()) != instance["sinks"]
+        or str(tuple(data["operators"]).__len__()) != instance["operators"]
+    ):
+        print("[Error] Test Zenoh-Flow configuration nodesSome nodes:  FAILED.")
```

Comment:
Considering the previous change I asked you to do, these lines need to be removed as well.